ui/src/app/components/workflows/workflows-home/workflows-home.component.html

```diff
@@ -0,0 +1,31 @@
+<clr-datagrid>
+
+  <clr-dg-action-bar>
+    <div class="btn-group">
+      <button type="button" class="btn btn-sm btn-secondary"><clr-icon shape="plus"></clr-icon> Create new workflow </button>
+    </div>
+  </clr-dg-action-bar>
+
+  <clr-dg-column>Workflow name</clr-dg-column>
+  <clr-dg-column>Project name</clr-dg-column>
+  <clr-dg-column>Is active</clr-dg-column>
+
+  <clr-dg-row *clrDgItems="let workflow of workflows" [clrDgItem]="workflow">
```

Comment:
Aren't these actions usually on the right side? I've seen three dots on the left side mostly to drag a row for reordering.

Reply:
Usually yes. In clarity no. No matter where I place action they are rendered on the left side.

Reply:
Hmm yes maybe they want to be consistent with the checkboxes (in batch action) which are usually on the left